### ISSUE TITLE
HTML Exporter detects a whole scene of text as one paragraph

### ISSUE THREAD

**Issue by AlanDunaway** [olivierkes/manuskript#972]
On 0.13.1, I tried to test the export functionality(Compile) and marked the HTML option(as I usually export HTML from my editor to Calibre to pack) and at the preview, when looking at the text, it has only one paragraph tag per scene(text), even though the text has paragraphs.
It should detect the paragraphs in the text and mark them with the tag.

**Comment by AlanDunaway** [comment]
It only detects a paragraph is there is a one line space between paragraphs.

**Comment by AlanDunaway** [comment]
Pandoc exporters also do this. Seems like a general issue with markdown.

**Comment by AlanDunaway** [comment]
There is a workaround that may help until a WYSIWYG editor is implemented:
When exporting, go to replace. Add a replacement(click the +), put \n in the Replace field and `<p>` in the WITH field.
That will replace all line breaks with paragraphs.

**Comment by AlanDunaway** [comment]
> There is a workaround that may help until a WYSIWYG editor is implemented: When exporting, go to replace. Add a replacement(click the +), put \n in the Replace field and `<p>` in the WITH field. That will replace all line breaks with paragraphs. This is just a workaround, it may bug with texts that use * * * as scene breaks.

This only work with in-program exports. Doesn't work with Pandoc exports.

**Comment by mxamber** [comment]
> It only detects a paragraph is there is a one line space between paragraphs. Pandoc exporters also do this. Seems like a general issue.

You're writing in Markdown. Markdown uses two line breaks as paragraph separator. That's like complaining HTML doesn't recognise paragraphs unless they're separated by `p` tags.

**Comment by AlanDunaway** [comment]
> > It only detects a paragraph is there is a one line space between paragraphs. Pandoc exporters also do this. Seems like a general issue.
> 
> You're writing in Markdown. Markdown uses two line breaks as paragraph separator. That's like complaining HTML doesn't recognise paragraphs unless they're separated by `p` tags. This is not a bug or an issue, that's how it's meant to work.

Yeah, I see I'm writing in Markdown due to the intricacies in formatting the text. But for me, Manuskript is way better for multiple book projects than yWriter(where I came from).
Now I have my ways to work around that, although I'll lose the easy formatting capabilities yWriter had.
I've set Calibre to convert from those to the tags I need to have and I do a bit of CSS formatting too.

**Comment by fish-face** [comment]
The bug here is that the editor formats single newlines as paragraph breaks, which is not true to markdown formatting - yet manuskript seems built around using markdown as a format language.
